### ISSUE TITLE
NPM commands are not working

### ISSUE THREAD

**Issue by arturopuente** [sergiodxa/contentz#35]
**Describe the bug**
Both the serve and build commands are failing.

When running either:

- `npm run dev:serve`
- `npm run dev:build`

Both commands fail because `serve` and `cli.js` are not included in devDependencies.

**To Reproduce**
1. Go to the project root directory
2. Run either `npm run dev:serve` or `npm run dev:build`

**Expected behavior**

Both commands should run in new Node environment without those packages installed.

**Desktop (please complete the following information):**
 - OS: macOS Mojave

**Comment by sergiodxa** [comment]
Yeah I was moving things as part of the refactor #18 and haven’t checked how to make it work in the mid time until it’s finished.
